src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -254,25 +253,25 @@ Amplify.configure(awsconfig);
 ```
 
 This will bootstrap your app to use Amplify with the deployed backend resources.
-
+ 
 **Recap:** You now have set up and configured your backend resources for Amplify authentication and are ready to start working on your front-end application.
 
 ## Determine your Auth integration path
 
 Before you implement Auth on your front-end application, you will want to evaluate the options of using the Authenticator UI component, Amplify Libraries methods, or Cognito Hosted UI for your use case. We recommend using Authenticator as the easiest to implement. You can also use Authenticator together with Amplify Libraries methods to extend capabilities when needed. Additionally, the Hosted UI is also best used with the Amplify Libraries because they provide a method to redirect to the Hosted UI.
```

Comment:
Small tweaks:

Before you implement Auth on your front-end application, you will want to evaluate the options of using the Authenticator connected UI component, using the Amplify Library APIs, or Cognito Hosted UI for your use case. We recommend using the Authenticator since it creates a sign in and registration experience for your with few lines of code. You can also use Authenticator together with Amplify Libraries methods to extend capabilities when needed.